src/Caching/Storages/FileStorage.php

```diff
@@ -249,6 +249,7 @@ public function clean(array $conditions): void
 	{
 		$all = !empty($conditions[Cache::ALL]);
 		$collector = empty($conditions);
+		$namespaces = $conditions[Cache::NAMESPACE] ?? false;
```

Comment:
Btw, probably better name is `NAMESPACES` to be consistent with `TAGS` and `FILES`